app/assets/stylesheets/general.scss

```diff
@@ -1,5 +1,10 @@
 @import "colors";
 
+html {
+	text-rendering: optimizeLegibility;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered -webkit-font-smoothing, text-rendering